.gitignore

```diff
@@ -1,5 +1,4 @@
-
-
+*.snakemake/
```

Comment:
This folder has a lot of local results from running Snakemake. I don't think we want to put it in github?